chia/_tests/wallet/did_wallet/test_did.py

```diff
@@ -889,6 +889,159 @@ async def test_did_transfer(self, self_hostname, two_wallet_nodes, with_recovery
             new_puzhash,
         )
 
+    @pytest.mark.parametrize(
+        "trusted",
+        [True, False],
+    )
+    @pytest.mark.anyio
+    async def test_did_auto_transfer_limit(self, self_hostname, two_wallet_nodes, trusted):
```

Comment:
I suppose this file is exempt from mypy checks. I think it would be good to add typing for new code anyway though.